server/chat.ts

```diff
@@ -1358,6 +1364,14 @@ export class CommandContext extends MessageContext {
 	requiresRoom() {
 		this.errorReply(`/${this.cmd} - must be used in a chat room, not a ${this.pmTarget ? "PM" : "console"}`);
 	}
+	commandDoesntExist(): never {
```

Comment:
```suggestion
	commandDoesNotExist(): never {
```

I don't like using combined words as parts of function names

Reply:
<_< All the same to me, I guess.